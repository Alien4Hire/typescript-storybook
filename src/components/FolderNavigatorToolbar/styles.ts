import { Box, Button } from '@mui/material';

import { styled } from '@mui/material/styles';

export const Divider = styled('div')(({ theme }) => ({
  borderRight: `1px solid ${theme.palette.gray.medium}`,
  height: '34px',
  margin: 'auto',
}));

export const Label = styled('div')(({ theme }) => ({
  fontWeight: '600',
  fontSize: '15px',
  letterSpacing: '0.4px',
  color: theme.palette.gray.veryDark,
  fontFamily: "'Source Sans Pro', sans-serif",
}));

export const Wrapper = styled('div')(({ theme }) => ({
  position: 'relative',
  background: theme.palette.gray.veryLight,
  paddingLeft: '56px',
}));

export const FolderForm = styled('div')`
  display: flex;
  align-items: center;
`;

export const FolderAdd = styled(Box, {
  shouldForwardProp: (prop: string) => prop !== 'active',
})(({ active, theme }) => ({
  position: 'absolute',
  top: 0,
  bottom: 0,
  left: 0,
  right: 0,
  width: 'auto',
  padding: '2px',
  backgroundColor: theme.palette?.gray.veryLight,
  display: 'none',
  ...(active && {
    display: 'flex',
    alignItems: 'center',
  }),
}));

export const CsvModalBackdrop = styled(Box)({
  display: 'flex',
  flexDirection: 'column',
  justifyContent: 'center',
  alignItems: 'center',
  backgroundColor: '#f1f1f1',
  borderRadius: '16px',
  padding: '32px 32px 70px',
});

export const UploadListItem = styled(Box)(({ theme }) => ({
  display: 'flex',
  alignItems: 'center',
  padding: '16px 0px',
  borderBottom: `1px solid ${theme.palette.gray.light}`,
}));

export const UploadButton = styled(Button)({
  textTransform: 'none',
  width: 'fit-content',
  marginTop: '16px',
});

export const DownloadTemplateWrapper = styled(Box)({
  display: 'flex',
  alignItems: 'center',
  marginTop: '24px',
  cursor: 'pointer',
});
