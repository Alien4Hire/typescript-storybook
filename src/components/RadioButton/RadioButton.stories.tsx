import React from 'react';
import { useArgs } from '@storybook/client-api';

import RadioButton from '.';
import { ComponentMeta, ComponentStory } from '@storybook/react';

export default {
  title: 'Component/RadioButton',
  component: RadioButton,
} as ComponentMeta<typeof RadioButton>;

const Template: ComponentStory<typeof RadioButton> = (args) => {
  const [, updateArgs] = useArgs();

  const handleChange = (
    e: React.ChangeEvent<HTMLInputElement>,
    checked: boolean
  ) => {
    updateArgs({ checked: args.value === e.target.value });
    args.onChange?.(e, checked);
  };

  return <RadioButton {...args} onChange={handleChange} />;
};

export const View = Template.bind({});
View.args = {
  label: 'label',
  value: '1',
  checked: false,
};
