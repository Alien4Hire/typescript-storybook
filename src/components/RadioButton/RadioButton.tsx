import React from 'react';
import { Radio, FormControlLabel, RadioProps } from '@mui/material';

import { ReactComponent as CheckboxChecked } from '../../icons/checkbox_checked.svg';

export type RadioButtonPropsT = RadioProps & {
  value?: string;
  label: string;
  testId?: string;
  ariaLabel?: string;
};

type RadioNativeProps = React.InputHTMLAttributes<HTMLInputElement> & {
  'data-testid'?: string;
};

const RadioButton = ({
  value,
  checked,
  onChange,
  className,
  label,
  testId = 'RadioButton',
  color,
  disabled = false,
  ariaLabel,
  ...rest
}: RadioButtonPropsT) => {
  return (
    <FormControlLabel
      control={
        <Radio
          value={value}
          checked={checked}
          className={className}
          size="small"
          inputProps={
            {
              'data-testid': testId,
              'aria-label': ariaLabel,
            } as RadioNativeProps
          }
          onChange={onChange}
          checkedIcon={<CheckboxChecked width={24} height={24} />}
          color={color}
          disabled={disabled}
          {...rest}
        />
      }
      label={label}
    />
  );
};

export default RadioButton;
