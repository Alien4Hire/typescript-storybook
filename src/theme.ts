import { createTheme } from '@mui/material/styles';
import mlColors from './util/ml-cdl-colors';
import {
  computerSmallMonitorMax,
  mobileScreenMax,
  mobileScreenMin,
  tabletScreenMax,
} from './util/breakpoints';

declare module '@mui/material/Typography' {
  interface TypographyPropsVariantOverrides {
    title: true;
    textLarge: true;
    textBasic: true;
    textSmall: true;
    textSub: true;
    buttonMedium: true;
    buttonSmall: true;
    buttonLarge: true;
  }
}

//UX variants definitions can be found at https://www.figma.com/file/bEonWYX4lesmqlFY6MqWCt/Course-Assembly-MVP?node-id=5540%3A249648
//Custom created variant names for typographies

declare module '@mui/material/styles/createTypography' {
  interface Typography {
    formTitle?: React.CSSProperties;
    formSubtitle?: React.CSSProperties;
    appTitle?: React.CSSProperties;
    title?: React.CSSProperties;
    textLarge?: React.CSSProperties;
    textBasic?: React.CSSProperties;
    textSmall?: React.CSSProperties;
    textSub?: React.CSSProperties;
    buttonMedium?: React.CSSProperties;
    buttonSmall?: React.CSSProperties;
    buttonLarge?: React.CSSProperties;
  }

  // allow configuration using `createMuiTheme`
  interface TypographyOptions {
    title?: React.CSSProperties;
    textLarge?: React.CSSProperties;
    textBasic?: React.CSSProperties;
    textSmall?: React.CSSProperties;
    textSub?: React.CSSProperties;
    buttonMedium?: React.CSSProperties;
    buttonSmall?: React.CSSProperties;
    buttonLarge?: React.CSSProperties;
  }
}

declare module '@mui/material/styles' {
  interface Theme {}
  interface ThemeOptions {}
  interface Palette {
    specials: Palette['primary'];
    aqua: Palette['primary'];
    black: Palette['primary'];
    blue: Palette['primary'];
    gray: Palette['primary'];
    green: Palette['primary'];
    magenta: Palette['primary'];
    orange: Palette['primary'];
    purple: Palette['primary'];
    red: Palette['primary'];
    teal: Palette['primary'];
    white: Palette['primary'];
    yellow: Palette['primary'];
  }
  interface PaletteOptions {
    specials: PaletteOptions['primary'];
    aqua: PaletteOptions['primary'];
    black: PaletteOptions['primary'];
    blue: PaletteOptions['primary'];
    gray: PaletteOptions['primary'];
    green: PaletteOptions['primary'];
    magenta: PaletteOptions['primary'];
    orange: PaletteOptions['primary'];
    purple: PaletteOptions['primary'];
    red: PaletteOptions['primary'];
    teal: PaletteOptions['primary'];
    white: PaletteOptions['primary'];
    yellow: PaletteOptions['primary'];
  }

  interface PaletteColor {
    macmillanRed?: string;
    offWhite?: string;
    veryLight?: string;
    medium?: string;
    veryDark?: string;
    fullPageDialogShadow?: string;
  }

  interface SimplePaletteColorOptions {
    macmillanRed?: string;
    offWhite?: string;
    veryLight?: string;
    medium?: string;
    veryDark?: string;
    fullPageDialogShadow?: string;
  }
}

export const theme = createTheme({
  palette: {
    background: {
      default: mlColors.off_white,
      paper: mlColors.pure_white,
    },
    primary: {
      main: mlColors.aqua,
    },
    secondary: {
      main: mlColors.aqua_dark,
    },
    specials: {
      main: mlColors.macmillan_red,
      macmillanRed: mlColors.macmillan_red,
      fullPageDialogShadow: 'rgba(0, 0, 0, 0.5)',
    },
    aqua: {
      main: mlColors.aqua,
      veryLight: mlColors.aqua_very_light,
      light: mlColors.aqua_light,
      medium: mlColors.aqua_medium,
      dark: mlColors.aqua_dark,
      veryDark: mlColors.dark_navy,
    },
    black: {
      main: mlColors.black,
    },
    blue: {
      main: mlColors.blue,
      veryLight: mlColors.blue_very_light,
      light: mlColors.blue_light,
      medium: mlColors.blue_medium,
      dark: mlColors.blue_dark,
    },
    gray: {
      main: mlColors.gray,
      veryLight: mlColors.gray_very_light,
      light: mlColors.gray_light,
      medium: mlColors.gray_medium,
      dark: mlColors.gray_dark,
      veryDark: mlColors.gray_very_dark,
    },
    green: {
      main: mlColors.green,
      veryLight: mlColors.green_very_light,
      light: mlColors.green_light,
      medium: mlColors.green_medium,
      dark: mlColors.green_dark,
    },
    magenta: {
      main: mlColors.magenta,
      veryLight: mlColors.magenta_very_light,
      light: mlColors.magenta_light,
      medium: mlColors.magenta_medium,
      dark: mlColors.magenta_dark,
    },
    orange: {
      main: mlColors.orange,
      veryLight: mlColors.orange_very_light,
      light: mlColors.orange_light,
      medium: mlColors.orange_medium,
      dark: mlColors.orange_dark,
    },
    purple: {
      main: mlColors.purple,
      veryLight: mlColors.purple_very_light,
      light: mlColors.purple_light,
      medium: mlColors.purple_medium,
      dark: mlColors.purple_dark,
    },
    red: {
      main: mlColors.red,
      veryLight: mlColors.red_very_light,
      light: mlColors.red_light,
      medium: mlColors.red_medium,
      dark: mlColors.red_dark,
    },
    teal: {
      main: mlColors.teal,
      veryLight: mlColors.teal_very_light,
      light: mlColors.teal_light,
      medium: mlColors.teal_medium,
      dark: mlColors.teal_dark,
      veryDark: mlColors.teal_very_dark,
    },
    white: {
      main: mlColors.pure_white,
      offWhite: mlColors.off_white,
    },
    yellow: {
      main: mlColors.yellow,
      veryLight: mlColors.yellow_very_light,
      light: mlColors.yellow_light,
      medium: mlColors.yellow_medium,
      dark: mlColors.yellow_dark,
    },
  },
  breakpoints: {
    values: {
      xs: 0,
      sm: mobileScreenMin,
      md: mobileScreenMax,
      lg: tabletScreenMax,
      xl: computerSmallMonitorMax,
    },
  },
  spacing: 8,
  typography: {
    fontSize: 14,
    fontFamily: [
      '-apple-system',
      'BlinkMacSystemFont',
      '"Segoe UI"',
      'Roboto',
      'Oxygen',
      'Ubuntu',
      'Cantarell',
      '"Fira Sans"',
      '"Droid Sans"',
      '"Helvetica Neue"',
      'Source Sans Pro',
      'sans-serif',
    ].join(','),
    allVariants: {
      fontWeight: 'normal',
    },
    button: {
      fontSize: 14,
      textTransform: 'uppercase',
    },
    h1: {
      color: mlColors.black,
      fontSize: 32,
      fontFamily: 'Source Sans Pro',
      fontWeight: 600,
    },
    h2: {
      color: mlColors.black,
      fontSize: 24,
      fontFamily: 'Source Sans Pro',
      fontWeight: 600,
    },
    h3: {
      color: mlColors.black,
      fontSize: 18,
      fontFamily: 'Source Sans Pro',
      fontWeight: 600,
    },
    h4: {
      color: mlColors.black,
      fontSize: 16,
      fontFamily: 'Source Sans Pro',
      fontWeight: 600,
    },
    title: {
      color: mlColors.black,
      fontSize: 28,
      fontFamily: 'Source Sans Pro',
      fontWeight: 200,
      fontStyle: 'normal',
    },
    textLarge: {
      color: mlColors.black,
      fontSize: 18,
      fontFamily: 'Source Sans Pro',
      fontWeight: 400,
      fontStyle: 'normal',
    },
    textBasic: {
      color: mlColors.black,
      fontSize: 16,
      fontFamily: 'Source Sans Pro',
      fontWeight: 400,
      fontStyle: 'normal',
    },
    textSmall: {
      color: mlColors.black,
      fontSize: 14,
      fontFamily: 'Source Sans Pro',
      fontWeight: 400,
      fontStyle: 'normal',
    },
    textSub: {
      color: mlColors.black,
      fontSize: 13,
      fontFamily: 'Source Sans Pro',
      fontWeight: 400,
      fontStyle: 'normal',
    },
    buttonLarge: {
      color: mlColors.black,
      fontSize: 15,
      fontFamily: 'Source Sans Pro',
      fontWeight: 400,
      fontStyle: 'normal',
    },
    buttonMedium: {
      color: mlColors.black,
      fontSize: 18,
      fontFamily: 'Source Sans Pro',
      fontWeight: 400,
      fontStyle: 'normal',
    },
    buttonSmall: {
      color: mlColors.black,
      fontSize: 12,
      fontFamily: 'Source Sans Pro',
      fontWeight: 400,
      fontStyle: 'normal',
    },
  },
  components: {
    MuiAlert: {
      styleOverrides: {
        filledError: {
          background: mlColors.red_very_light,
          color: mlColors.black,
          border: `1px solid ${mlColors.red}`,
        },
        filledInfo: {
          background: mlColors.blue_very_light,
          color: mlColors.black,
          border: `1px solid ${mlColors.blue}`,
        },
        filledSuccess: {
          background: mlColors.green_very_light,
          color: mlColors.black,
          border: `1px solid ${mlColors.green}`,
        },
        filledWarning: {
          background: mlColors.yellow_very_light,
          color: mlColors.black,
          border: `1px solid ${mlColors.yellow}`,
        },
      },
    },
    // MuiButton: {
    //   styleOverrides: {
    //     root: {
    //       fontSize: '1rem',
    //       textTransform: 'none',
    //       height: '35px',
    //       padding: '0 15px',
    //       margin: '5px',
    //       '&:focus': {
    //         boxShadow: '0 0 6px rgba(0, 117, 142, 1)',
    //         border: '1px solid #FFFFFF',
    //       },
    //       '&:disabled': {
    //         background: '#c8c8c8',
    //         color: '#f3f3f3',
    //       },
    //     },
    //     textSecondary: {
    //       fontSize: '0.9375rem',
    //       height: '34px',
    //       lineHeight: '33px',
    //       padding: '0 15px',
    //       borderRadius: '3px',
    //       '-webkit-border-radius': '3px',
    //       '-ms-border-radius': '3px',
    //       '-moz-border-radius': '3px',
    //       textDecoration: 'none',
    //       color: '#00758e',
    //       cursor: 'pointer',
    //       background: 'transparent',
    //       border: '1px solid #00758e',
    //       overflow: 'hidden',
    //     },
    //     containedPrimary: {
    //       '&:focus': {
    //         boxShadow: '0 0 6px rgba(0, 117, 142, 1)',
    //         border: '1px solid #FFFFFF',
    //       },
    //       '&:disabled': {
    //         background: '#c8c8c8',
    //         color: '#f3f3f3',
    //       },
    //     },
    //     outlinedSecondary: {
    //       '&:focus': {
    //         boxShadow: '0 0 6px rgba(0, 117, 142, 1)',
    //         border: '1px solid #00758e',
    //       },
    //       '&:disabled': {
    //         color: '#989898',
    //         fill: '#c8c8c8',
    //         backgroundColor: '#f3f3f3',
    //       },
    //     },
    //   },
    // },
    MuiButtonBase: {
      defaultProps: {
        disableRipple: true, // No more ripple, on the whole application!
        focusRipple: false,
      },
    },
  },
});
