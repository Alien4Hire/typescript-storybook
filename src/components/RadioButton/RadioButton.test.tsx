import { render, fireEvent, screen } from '@testing-library/react';
import { ThemeProvider, StyledEngineProvider } from '@mui/material/styles';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import { theme } from '../../theme';
import '../../setupTests';

import RadioButton, { RadioButtonPropsT } from './RadioButton';

beforeAll(() => {
  window.getComputedStyle = jest.fn();
});

const baseProps = {
  label: 'Test',
  value: 'test value',
  onChange: jest.fn(),
  checked: false,
  testId: 'testId',
};

const Component = (props: RadioButtonPropsT) => (
  <StyledEngineProvider injectFirst>
    <ThemeProvider theme={theme}>
      <RadioButton {...props} />
    </ThemeProvider>
  </StyledEngineProvider>
);

const renderComponent = (customProps = {}) => {
  const props: RadioButtonPropsT = { ...baseProps, ...customProps };

  return render(<Component {...props} />);
};

describe('<RadioButton />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Component {...baseProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render correct label', () => {
    renderComponent();
    expect(screen.getByText(baseProps.label)).toBeInTheDocument();
  });

  describe('check onChange fn', () => {
    it('should call function after checkbox click', () => {
      renderComponent();
      const radioElement: HTMLInputElement = screen.getByLabelText('Test');

      expect(radioElement.checked).toBe(false);
      expect(baseProps.onChange).not.toBeCalled();

      fireEvent.click(radioElement);
      expect(baseProps.onChange).toBeCalledTimes(1);
    });

    it('should call function after label click', () => {
      renderComponent();
      const radioLabel = screen.getByLabelText('Test');

      fireEvent.click(radioLabel);
      expect(baseProps.onChange).toBeCalledTimes(1);
    });
  });
});
