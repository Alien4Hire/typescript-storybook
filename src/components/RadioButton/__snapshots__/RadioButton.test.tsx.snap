// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RadioButton /> renders correctly 1`] = `
.c6 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: fill 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  font-size: 1.25rem;
}

.c1 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
}

.c1::-moz-focus-inner {
  border-style: none;
}

.c1.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

.c9 {
  margin: 0;
  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Fira Sans","Droid Sans","Helvetica Neue",Source Sans Pro,sans-serif;
  font-weight: normal;
  font-size: 1rem;
  line-height: 1.5;
}

.c2 {
  padding: 9px;
  border-radius: 50%;
}

.c4 {
  cursor: inherit;
  position: absolute;
  opacity: 0;
  width: 100%;
  height: 100%;
  top: 0;
  left: 0;
  margin: 0;
  padding: 0;
  z-index: 1;
}

.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  cursor: pointer;
  vertical-align: middle;
  -webkit-tap-highlight-color: transparent;
  margin-left: -11px;
  margin-right: 16px;
}

.c0.Mui-disabled {
  cursor: default;
}

.c0 .MuiFormControlLabel-label.Mui-disabled {
  color: rgba(0,0,0,0.38);
}

.c5 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c7 {
  -webkit-transform: scale(1);
  -ms-transform: scale(1);
  transform: scale(1);
}

.c8 {
  left: 0;
  position: absolute;
  -webkit-transform: scale(0);
  -ms-transform: scale(0);
  transform: scale(0);
  -webkit-transition: -webkit-transform 150ms cubic-bezier(0.4,0,1,1) 0ms;
  -webkit-transition: transform 150ms cubic-bezier(0.4,0,1,1) 0ms;
  transition: transform 150ms cubic-bezier(0.4,0,1,1) 0ms;
}

.c3 {
  color: rgba(0,0,0,0.6);
}

.c3:hover {
  background-color: rgba(0,117,142,0.04);
}

.c3.Mui-checked {
  color: #00758e;
}

.c3.Mui-disabled {
  color: rgba(0,0,0,0.26);
}

@media print {
  .c1 {
    color-adjust: exact;
  }
}

@media (hover:none) {
  .c3:hover {
    background-color: transparent;
  }
}

<label
  className="c0 MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd"
>
  <span
    className="c1 MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root c2 MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root c3"
    onBlur={[Function]}
    onContextMenu={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={null}
  >
    <input
      checked={false}
      className="c4 PrivateSwitchBase-input"
      data-testid="testId"
      disabled={false}
      onChange={[Function]}
      type="radio"
      value="test value"
    />
    <span
      className="c5"
    >
      <svg
        aria-hidden={true}
        className="c6 MuiSvgIcon-root MuiSvgIcon-fontSizeSmall c7"
        data-testid="RadioButtonUncheckedIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
        />
      </svg>
      <svg
        aria-hidden={true}
        className="c6 MuiSvgIcon-root MuiSvgIcon-fontSizeSmall c8"
        data-testid="RadioButtonCheckedIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
        />
      </svg>
    </span>
  </span>
  <span
    className="c9 MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label"
  >
    Test
  </span>
</label>
`;
